src/rmodels.c

```diff
@@ -5106,6 +5166,164 @@ static Model LoadGLTF(const char *fileName)
 
     return model;
 }
+
+static void GetGLTFPoseAtTime(cgltf_accessor* input, cgltf_accessor *output, float time, void *data) {
+    assert(input->count == output->count);
+
+    float tstart = 0;
+    float tend = 0;
+
+    int keyframe = 0; // defaults to first pose
+    for(int i = 0; i < input->count - 1; i++) {
```

Comment:
minor optimization note: `i` starts from zero every time this function is called, but since we're stepping through at regular intervals (i.e. time = frame * GLTF_ANIMATION_DELAY), some additional bookkeeping could be added to track the previous `i` for `GetGLTFPoseAtTime`. Since this is just for constructing the initial poses, I didn't bother since this was simpler and I'm assuming most animations will be relatively simple.